src/main/java/org/zeromq/ZMQ.java

```diff
@@ -797,9 +810,21 @@ public void close()
          *
          * @return the socket type.
          */
-        public SocketType getType()
```

Comment:
I broke this API and I'm sorry, thank you for correcting this. :+1:

Reply:
Neh, no problem, no need to say sorry :)